app/gui/src/controller/visualization/manager.rs

```diff
@@ -285,11 +286,15 @@ impl Manager {
             // Early return: requested to remove visualization that was already removed.
             return;
         };
+        let prj = self.executed_graph.module_qualified_name(&*self.executed_graph.project);
```

Comment:
This is the line that requires [project to be public](https://github.com/enso-org/enso/pull/7115#discussion_r1242031934).

Reply:
I think executed graph's `module_qualified_name` (https://github.com/enso-org/enso/blob/3e6af201dbecffdedd3b6dffde7d51957bfcec9a/app/gui/src/controller/graph/executed.rs) should not take project as an argument, but take the project it owns instead.

Reply:
Done in [429001e](https://github.com/enso-org/enso/pull/7115/commits/429001e689408a191dd952b4e746a9caa8fd58c4)